### ISSUE TITLE
AutoSmurf - How do you recycle Toon Names?

### ISSUE THREAD

**Issue by AbbyRoad** [kolton/d2bot-with-kolbot#1958]
Hi

I'm new at this.  I have the stock version of AutoSmurf working.  My four characters made it into Hell level and then I deleted and then recreated the same characters so I could restart from zero in Normal.  Unfortunately, instead of doing the quests, the Sorc (leader) decides to try to get all of the waypoints versus restarting all of the quests.  (My first hint of me missing something was when the program informed me of the amount of XP my four Toons had lost on entering the game.)

So, my question is - what do I need to delete from the bot files to recycle the Toon names and restart from zero?

Thanks

AbbyRoad

**Comment by mf022** [comment]
Did you change the difficulty to normal in the profiles?

Maybe you should delete the *.json files from ...\kolbot\data\ and ...\kolbot\logs\

Only for d2bot-with-kolbot maintenance *.txt files from ...\kolbot\data\secure\ should be deleted from time to time.

**Comment by AbbyRoad** [comment]
Hi mf022

I had the profiles set to "highest", so I switched them to "normal", but no change in actions. Deleting the .json files had no impact.

All members enter game, party up.  The leader goes to the WP in town, and puts the following message above her head - "Trying to get the WP".  She then marches out of town and gets WPs til she dies.  The rest of the team stays in town.

I tried changing the profile names and the toon names, and the same thing happens, so I guess it is beyond recycling toon names.  :)

I have the two Bot folders on the desk top.  The first is for running Normal and Nightmare, and the second is for transitioning to and then running Hell.  This did not cause any issues when I transitioned "up in difficulty", but could it be causing issues now?  I don't see how as the folder contents are being used independently, but I'm not a skilled programmer.

Regards;

AbbyRoad
